### ISSUE TITLE
shift_logits

### ISSUE THREAD

**Issue by ASENNIU** [yangjianxin1/ClipCap-Chinese#15]
作者你好，我是nlp初学者，请问为什么在计算交叉熵的时候要shift logits喃。
hift_logits = logits[..., args.prefix_len - 1:-1, :].contiguous().view(-1, logits.size(-1))
